css-layout-classes/styles.css

```diff
@@ -0,0 +1,43 @@
+div.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+div.row {
+  display: flex;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-two-thirds {
+  flex-basis: calc(2 / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+h1 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 2rem;
+}
+
+h2 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1.25rem;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 0.95rem;
+}
+
+#kirby,
+#fox,
+#pikachu,
+#ness {
```

Comment:
max width is a limiting value, not a set value. It would be best to use `width:100%` here. Also, you need to make sure that width is applied to all img elements, not just these four